src/scripts/main.js

```diff
@@ -1 +1,91 @@
 'use strict';
+
+let isClickedLeft = false;
+let isClickedRight = false;
+let timer;
+const notification = document.createElement('div');
+
+notification.setAttribute('data-qa', 'notification');
+
+function resetTimer() {
+  clearTimeout(timer);
+
+  timer = setTimeout(() => {
+    if (!isClickedLeft) {
+      firstPromise();
+    }
+  }, 3000);
+}
+
+document.addEventListener('mousedown', (e) => {
```

Comment:
You're calling `firstPromise` and `resetTimer` every time the left mouse button is clicked. This could lead to multiple notifications being shown and the timer being reset before the 3 seconds have passed. If this is not the intended behavior, you might want to revise this logic.